evm/rlp/transactions.py

```diff
@@ -160,3 +160,32 @@ class BaseShardingTransaction(rlp.Serializable):
     @property
     def hash(self):
         return keccak(rlp.encode(self))
+    
+    @property
+    def intrensic_gas(self):
```

Comment:
*organization nitpick*

This seems like it should be next to the `get_intrensic_gas` method.